### PR TITLE
RHDEVDOCS-4301 - Logging 5.3.12, 5.4.6, 5.5.2 Release Notes

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.9+

Issue: https://issues.redhat.com/browse/RHDEVDOCS-4301 

Link to docs preview: https://drive.google.com/file/d/1JqPbFD9Jb9NMyQ83_0r0nbGhYgQXr81Q/view?usp=sharing

Reviewers: 
@anpingli / @kabirbhartiRH  / @QiaolingTang 
@jcantrill / @alanconway

### CODE REVIEW

**Review by jcantrill — COMMENTED** [review]
/lgtm

File: modules/cluster-logging-rn-5.5.2.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+* Before this update, if you deleted the Kibana Custom Resource, the {product-title} web console continued displaying a link to Kibana. With this update, removing the Kibana Custom Resource also removes that link. (link:https://issues.redhat.com/browse/LOG-3053[LOG-3053])
+
+* Before this update, each rollover job created empty indices when the `ClusterLogForwarder` custom resource had JSON parsing defined. With this update, new indices are not empty. (link:https://issues.redhat.com/browse/LOG-3063[LOG-3063])
+
+* Before this update, resources created by a prior version of the Loki Operator remained after upgrading to 5.5. With this update, upgrading the Operator cleans up resources correctly. (link:https://issues.redhat.com/browse/LOG-2945[LOG-2945])
+
+* Before this update, a user was not able to view the application logs of namespaces they have access to. With this update, the Loki Operator automatically creates a cluster role and cluster role binding allowing users to read application logs. (link:https://issues.redhat.com/browse/LOG-2918[LOG-2918])
+
```
Comment (anpingli) [inline modules/cluster-logging-rn-5.5.2.adoc:32]: There are two new bugs. Could add them?   The other part looks good.
https://issues.redhat.com/browse/LOG-2970
https://issues.redhat.com/browse/LOG-2957

Comment (libander) [reply modules/cluster-logging-rn-5.5.2.adoc:32]: https://issues.redhat.com/browse/LOG-2957 is flagged no-rn....

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**